tsconfig.json

```diff
@@ -1,22 +1,27 @@
 {
   "compilerOptions": {
-    "baseUrl": "src",
-    "rootDir": "./",
     "outDir": "build",
-    "target": "esnext",
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "declaration": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
     "module": "commonjs",
     "moduleResolution": "node",
-    "sourceMap": true,
-    "removeComments": true,
-    "strict": true,
     "resolveJsonModule": true,
-    "esModuleInterop": true,
-    "allowJs": true,
-    "declaration": true,
+    "isolatedModules": true,
+    "sourceMap": true,
     "typeRoots": ["./src/types", "./node_modules/@types"]
   },
   "include": [
-    "src/**/*",
-    "test/**/*"
```

Comment:
Why did we removethe test directory?

Reply:
There is no need to have it in the includes in the tsconfig.json compile options it is linted through the tsconfig.eslint.json